code/script.js

```diff
@@ -0,0 +1,88 @@
+let USER = 'qwer-ctrl'
+
+const USER_URL = `https://api.github.com/users/${USER}`
+const REPOS_URL = `https://api.github.com/users/${USER}/repos`
+
+
+const options = {
+    method: "GET", 
+    headers: {
+        Authorization: `token ${API_TOKEN}`,
+    }
+}
+console.log(API_TOKEN)
+const profile = document.getElementById('profile-container')
+const projects = document.getElementById('projects')
+
+const fetchProfile = () => {
+fetch(USER_URL, options)
+    .then((response) => response.json())
+    .then((data) => {
+     profile.innerHTML += `
+        <div id="profile-container" class="profile">
+        <div class="profile-image">
+        <a href="${USER_URL}">
+            <img src="${data.avatar_url}" alt="Avatar of ${data.login}">
+            </a>
+            </div>
+            <p class="username">${data.login}<p>
+        </div>
+        <div>
+        <p class="public-repositories">Public repositories: ${data.public_repos}</p>
+        </div>
```

Comment:
Look through the indentations here so the opening and closing-tag are at the same indentation. Also,  indent the "childs".